code_challenges.rb

```diff
@@ -1,19 +1,81 @@
-# ASSESSMENT 4: Ruby Coding Practical Questions
+   # ASSESSMENT 4: Ruby Coding Practical Questions
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
 
-
-
-
-
 fullArr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
 
 
+# def oddSort array
+#     newArr1= []
+#     array.filter do |element|
+#         if element.is_a? Integer
+#             newArr1<< element
+#         end 
+#     end 
+#     array = []
+#     newArr1.filter do |newArr1|
+#         if newArr1%2 === 1
+#             array << newArr1
+#         end 
+#     end 
+#     print array.sort 
+#     puts ""
```

Comment:
woo! you can also use `p` instead of `print`, so you can eliminate the extra `puts ""`